client/README.md

```diff
@@ -0,0 +1,42 @@
+# Wagtail client-side components
+
+This library aims to give developers the ability to subclass and configure Wagtail's UI components.
+
+## Usage
+
+```
+npm install wagtail
+```
+
+```javascript
+import { Explorer } from 'wagtail';
+
+...
+
+<Explorer onChoosePage={(page)=> { console.log(`You picked ${page}`); }} />
+
+```
+
```

Comment:
@jonnyscholes that's not really a blocker for merging, feel free to write something if you want :)